File: python/benchmark/bench_context.py
import yaml

ydoc = yaml.load(open('bench_context.yaml'))

class StockInfo:
    def __init__(self, name, name2, url, symbol, price, change, ratio):
        self.name   = name
        self.name2  = name2
        self.url    = url
        self.symbol = symbol
        self.price  = price
        self.change = change
        self.ratio  = ratio

list = [StockInfo(**hash) for hash in ydoc['list']]

locals().pop('yaml')
locals().pop('ydoc')
locals().pop('hash')
locals().pop('StockInfo')

File: ruby/benchmark/bench_context.rb
require 'yaml'

class StockInfo
  def initialize(values={})
    @name   = values['name']
    @name2  = values['name2']
    @url    = values['url']
    @symbol = values['symbol']
    @price  = values['price']
    @change = values['change']
    @ratio  = values['ratio']
  end
  attr_accessor :name, :name2, :url, :symbol, :price, :change, :ratio
end

ydoc = YAML.load_file('bench_context.yaml')
@list = ydoc['list'].collect { |values| StockInfo.new(values) }
